src/nbl/video/device_capabilities/device_limits.json

```diff
@@ -1442,7 +1440,7 @@
         {
             "type": "uint32_t",
             "name": "minImportedHostPointerAlignment",
-            "value": "0x1u << 31",
+            "value": "0x1u << 14u",
             "compare": "REVERSE",
             "comment": [
```

Comment:
do we require external memory host extension?

Also we probably shouldn't in-case we do WebGPU in the future.

so thats the reason the alignment is max uint32_t Power of two, so you can't fail limit intersection when extension/feature not enabled

Also its a good worst case value to report

Reply:
only when renderdoc is used

Reply:
ok then don't change